packages/lab/src/badge/Badge.css

```diff
@@ -1,67 +1,33 @@
-/* Default variables applied to the root element */
-.saltBadge {
-  /* Colors */
-  --badge-content-text-color: var(--saltBadge-content-text-color, var(--salt-accent-foreground));
-  --badge-content-icon-color: var(--saltBadge-content-icon-color, var(--salt-accent-background));
-  /* Font, Typography */
-  --badge-content-borderRadius: 16px;
-  --badge-content-lineHeight: 15px;
-  --badge-content-padding: 0px 4px;
-  --badge-fontSize: var(--saltBadge-fontSize, var(--salt-text-label-fontSize));
-
-  /* Derived */
-  /* Same as content padding in x */
-  --badge-offset-inline: -4px;
-
-  --badge-sequence-gap: var(--salt-size-unit);
-}
-
 /* Style applied to the root element */
 .saltBadge {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   display: inline-flex;
-  position: relative;
   flex-shrink: 0;
-  vertical-align: middle;
-  line-height: var(--salt-text-lineHeight);
-  font-size: var(--salt-text-fontSize);
-}
-
-/* Give automatic margin when they sit next to each other */
-.saltBadge + .saltBadge {
-  margin-left: var(--badge-sequence-gap);
+  justify-content: center;
 }
 
-/* Style applied to the badge in the corner of the node */
 .saltBadge-badge {
-  color: var(--badge-content-text-color);
-  display: inline-block;
-  font-weight: 700; /* TODO: Use characteristic once aligned */
-  font-size: var(--badge-fontSize);
-  /* default small type */
-  width: auto;
-  min-width: 15px;
-  line-height: var(--badge-content-lineHeight);
-  border-radius: var(--badge-content-borderRadius);
-  text-align: center;
+  /* Should this vary according to touch size */
```

Comment:
is there a decision on this?

Reply:
nope spacing is still pending a decision from design